### PR TITLE
Add heroku config on demand

### PR DESCRIPTION
**Pull request by ghn**

### CODE REVIEW

File: {{cookiecutter.project_slug}}/release-tasks.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+python manage.py migrate --noinput
```
Comment (sephii) [inline {{cookiecutter.project_slug}}/release-tasks.sh]: Put this in the `scripts` directory.

File: {{cookiecutter.project_slug}}/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+-r requirements/heroku.in
```
Comment (sephii) [inline {{cookiecutter.project_slug}}/requirements.txt]: This should include the pinned version of the requirements file, aka `requirements/heroku.txt` (it doesn't exist but will be created by Drifter).

**Review by sephii — CHANGES_REQUESTED** [review]

File: {{cookiecutter.project_slug}}/release-tasks.sh (hunk 1 above)
Comment (ghn) [reply {{cookiecutter.project_slug}}/release-tasks.sh]: done

File: {{cookiecutter.project_slug}}/requirements.txt (hunk 2 above)
Comment (ghn) [reply {{cookiecutter.project_slug}}/requirements.txt]: done

**Comment by ghn** [comment]
base branch?? -> close

**Pull request closed without merging**